policy/modules/system/init.te

```diff
@@ -541,6 +542,11 @@ ifdef(`init_systemd',`
 		files_mounton_non_security(init_t)
 	')
 
+	optional_policy(`
+		# to run systemd-boot-update
+		bootloader_domtrans(init_t)
+	')
```

Comment:
I'm torn on having this vs. `init_system_domain()` in bootloader. I'm concerned about potential NNP issues.

Reply:
The only thing that is needed is for systemd to start `systemd-boot-update.service` which just runs `bootctl --no-variables --graceful update`. `bootctl` itself should only be on the root filesystem so I'm not sure if NNP will be an issue. Am I missing something?